odoo_module_migrate/migration_scripts/migrate_120_130.py

```diff
@@ -2,6 +2,22 @@
 # @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
+_TEXT_WARNINGS = {
+    ".xml": {
+        r".*active.*toggle_button":
+            'You can remove actions archive/unarchive',
+        r".*active.*boolean_button":
+            'You can remove actions archive/unarchive'
```

Comment:
Proper regexp  would be ``r'active.*widget="toggle_button"|widget="toggle_button".*active|widget="boolean_button".*active|active.*widget="boolean_button"'``